src/main/resources-filtered/org/obsidiantoaster/generator/ui/starter/plugin-template.xml.ftl

```diff
@@ -10,7 +10,7 @@
          <plugin>
             <groupId>io.fabric8</groupId>
             <artifactId>fabric8-maven-plugin</artifactId>
-            <version>3.1.92</version>
+            <version>${version.fabric8-maven-plugin}</version>
```

Comment:
i.e. at runtime the forge addon uses what gets generated into `target/classes` which has no maven properties any more